msamp/operators/fp8_op/fp8_op.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Fp8Op module."""
+
+import ctypes
+
+
+class Fp8Op:
+    """Msampfp8 library wrapper class."""
+    lib = ctypes.cdll.LoadLibrary('libmsampfp8.so')
```

Comment:
pls check its presence before loading and add error handlings
do you want to use the absolute path in case user installs on host?

Reply:
Good suggestion. Added logic to check presence before loading and use absolute path for the msamp fp8 library.